official/keras/train.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import print_function
+
+import argparse
+import os
+import sys
+
+import cifar10_dataset
+import keras
+from keras.callbacks import ModelCheckpoint
+import resnet50_cifar10_model
+import tensorflow as tf
+
+# Training parameters
+parser = argparse.ArgumentParser()
+
+parser.add_argument(
+    '--batch_size',
+    type=int,
+    default=32,
+    help='Number of images to process in a batch')
+
+parser.add_argument(
+    '--epochs',
+    type=int,
+    default=2,
+    help='Number of training steps')
+
+def train(resnet50_cifar10_model, train_dataset, test_dataset, optimizer, loss_fn):
+  train_dataset = train_dataset.batch(FLAGS.batch_size)
+  test_dataset = test_dataset.batch(FLAGS.batch_size)
+
+  (train_images, train_labels) = train_dataset.make_one_shot_iterator().get_next()
+  train_images = tf.cast(train_images, tf.float32)
+  train_labels = tf.cast(train_labels, tf.float32)
+  (test_images, test_labels) = test_dataset.make_one_shot_iterator().get_next()
+  test_images = tf.cast(test_images, tf.float32)
+  test_labels = tf.cast(test_labels, tf.float32)
+
+  model = resnet50_model(train_images)
```

Comment:
Before defining the model, set the learning phase to `True` (and maybe set `image_data_format` to `channels_first`, or make it configurable)